arviz/stats/stats.py

```diff
@@ -464,7 +464,7 @@ def psislw(log_weights, reff=1.0):
         xcutoff = max(x[x_sort_ind[cutoff_ind]], cutoffmin)
 
         expxcutoff = np.exp(xcutoff)
-        tailinds, = np.where(x > xcutoff)
+        tailinds, *_ = np.where(x > xcutoff)
```

Comment:
np.flatnonzero?

Reply:
This one I was confused about. Should I just add pylint or was it giving the correct message?

And in regards to your comments how does `np.flatnonzero` help here? Looked at the docs but couldn't understand
https://docs.scipy.org/doc/numpy/reference/generated/numpy.flatnonzero.html

Reply:
Changed back to original statement

Reply:
Your solution looks good